hyperscale/009-other_grants.sql

```diff
@@ -0,0 +1,18 @@
+GRANT EXECUTE ON FUNCTION create_extension(text) TO citus WITH GRANT OPTION;
+GRANT EXECUTE ON FUNCTION drop_extension(text) TO citus WITH GRANT OPTION;
+GRANT ALL ON SCHEMA public TO citus WITH GRANT OPTION;
+GRANT SELECT ON pg_buffercache TO public;
+GRANT EXECUTE ON FUNCTION pg_buffercache_pages() TO public;
+GRANT pg_monitor TO citus;
+GRANT USAGE ON SCHEMA cron TO citus WITH GRANT OPTION;
+GRANT ALL ON SCHEMA partman TO citus WITH GRANT OPTION;
+GRANT ALL ON ALL TABLES IN SCHEMA partman TO citus WITH GRANT OPTION;
+GRANT EXECUTE ON ALL FUNCTIONS IN SCHEMA partman TO citus WITH GRANT OPTION;
+GRANT EXECUTE ON ALL PROCEDURES IN SCHEMA partman TO citus WITH GRANT OPTION;
+GRANT USAGE ON FOREIGN DATA WRAPPER dblink_fdw TO citus WITH GRANT OPTION;
+GRANT EXECUTE ON FUNCTION public.pg_replication_origin_create(text) TO citus;
+GRANT EXECUTE ON FUNCTION public.pg_replication_origin_session_progress(bool) TO citus;
+GRANT EXECUTE ON FUNCTION public.pg_replication_origin_progress(text, bool) TO citus;
+GRANT EXECUTE ON FUNCTION public.pg_replication_origin_drop(text) TO citus;
+GRANT EXECUTE ON FUNCTION public.pg_replication_origin_xact_setup(pg_lsn, timestamptz) TO citus;
+GRANT EXECUTE ON FUNCTION public.pg_replication_origin_session_setup(text) TO citus;
```

Comment:
The last 6 grants are already in hyperscale/004-postgres_replication_origins.sql

Reply:
Removed